src/content/data/ua_overrides.jsm

```diff
@@ -54,6 +54,22 @@ const UAOverrides = [
       let prefix = originalUA.substr(0, originalUA.indexOf(")") + 1);
       return `${prefix} AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.98 Safari/537.36`;
     }
+  },
+
+  /*
+   * WebCompat bug #16401
+   * https://webcompat.com/issues/16401
+   *
+   * Starting with Firefox 61, the window.controllers shim will no longer be
+   * available, causing ib.absa.co.za to show a "Browser unsupported" message
+   * and blocking the page.
+   */
+  {
+    baseDomain: "absa.co.za",
+    uriMatcher: (uri) => uri.includes("ib.absa.co.za"),
+    uaTransformer: (originalUA) => {
+      return `${originalUA} AppleWebKit/605.1.15 for Compatibility`;
```

Comment:
The `for Compatibility` bit is interesting! I wonder if we shouldn't have it in general though, as it might confuse certain UA parsing libs? Thoughts @wisniewskit?

Reply:
@denschub can you remind me the status of console logging a message for the site developers?

Reply:
We currently end up in the browser console. We have bug 1316668, but that didn't make it into v2 in our triage some months ago.